.travis.yml

```diff
@@ -3,26 +3,23 @@ language: ruby
 services:
   - docker
 rvm:
-  - 2.2.10
   - 2.3.8
-  - 2.4.6
-  - 2.5.5
-  - 2.6.3
+  - 2.4.10
```

Comment:
the Ruby core community [dropped support](https://www.ruby-lang.org/en/downloads/branches/) for versions below 2.6, I'm thinking we should drop any versions below 2.6, as well.

Reply:
I would prefer to continue to support these versions as long as it is not inconvenient. On the off chance that someone needs a feature from this version, but they find it difficult to upgrade their Ruby version, I want to be accommodating to their needs.

Reply:
Debian oldstable, still in lts, is ruby 2.3.3.

Reply:
_wowwwwwwwww_